packages/udes-ui-validation/src/test/component.test.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { cleanup, render, fireEvent } from "@testing-library/react";
+import MaterialComponent from "../exampleComponent/materialUiComponent";
+
+afterEach(cleanup);
+
+describe("Validator Material UI tests", () => {
+  test("Valid state causes no change", async () => {
```

Comment:
The name of this test confused me. Isn't it just checking you can input some valid entries and not get an error and input cleared? No change to me sounds like the input field won't change.

Reply:
When there is an error there is a proper change that is implemented by material-ui: it makes new divs. Happy to take suggestions on another name

Reply:
Yep, I imagine for most non simpletons this name is fine. I have no better naming solution- so happy for you to leave it, the test is clear what it does.